gradle/libs.versions.toml

```diff
@@ -1,6 +1,7 @@
 [versions]
+micronaut = "4.1.1"
```

Comment:
please, lets use `micronaut` for core and `micronaut-platform` for platform. We do that in every other repository.

Reply:
We do?

Hmmm... test suite uses the library plugin so picks up the "micronaut" version...

I can change it, but it feels weird for the correct way to be more code.

Reply:
we do yes. `micronaut` means micronaut core in every repo.

Reply:
Security:
https://github.com/micronaut-projects/micronaut-security/blob/master/gradle/libs.versions.toml#L4-L5

R2DBC
https://github.com/micronaut-projects/micronaut-r2dbc/blob/master/gradle/libs.versions.toml#L2-L3

Serde: 
https://github.com/micronaut-projects/micronaut-serialization/blob/master/gradle/libs.versions.toml#L2-L3

Kafka: 

https://github.com/micronaut-projects/micronaut-kafka/blob/master/gradle/libs.versions.toml#L2-L3

Reply:
Ahhh...is it https://github.com/micronaut-projects/micronaut-kafka/blob/master/buildSrc/src/main/groovy/io.micronaut.internal.build.kafka-testsuite.gradle#L26 that stops it failing?